src/components/application_manager/include/application_manager/policies/policy_handler.h

```diff
@@ -901,8 +901,31 @@ class PolicyHandler : public PolicyHandlerInterface,
    */
   void GetRegisteredLinks(std::map<std::string, std::string>& out_links) const;
 
+  /**
+   * @brief Load policy manager
+   * This method is thread safe
+   * @return Pointer to the policy manager instance or null if not inited
+   */
+  std::shared_ptr<PolicyManager> LoadPolicyManager() const;
+
+  /**
+   * @brief Exchange a policy manager
+   * This method is thread safe
+   * @param policy_manager - new policy manager
+   * @return Pointer to the previous policy manager
+   * or null if the policy manager not set previously
+   */
+  std::shared_ptr<PolicyManager> ExchangePolicyManager(
```

Comment:
Looks like return value is redundant, isn't it?

Reply:
@LuxoftAKutsan Return value was removed